File: .NET/Microsoft.Recognizers.Text.DataTypes.TimexExpression/TimexRelativeConvert.cs
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

namespace Microsoft.Recognizers.Text.DataTypes.TimexExpression
{
    public static class TimexRelativeConvert
    {
        public static string ConvertTimexToStringRelative(TimexProperty timex, System.DateTime referenceDate)
        {
            return TimexRelativeConvertEnglish.ConvertTimexToStringRelative(timex, referenceDate);
        }
    }
}

File: Python/libraries/datatypes-timex-expression/datatypes_timex_expression/timex_relative_convert.py

class TimexRelativeConvert:

    @staticmethod
    def convert_timex_to_string_relative(timex, reference_date):
        from .english import english_convert_timex_to_string_relative
        return english_convert_timex_to_string_relative(timex=timex, date=reference_date)
